packages/utils/src/lib/git.integration.test.ts

```diff
@@ -21,12 +30,19 @@ describe('git utils', () => {
 
     await git.add('README.md');
     await git.commit('Create README');
+
+    await git.checkout(['-b', 'feature-branch']);
   });
 
   afterAll(async () => {
     await rm(baseDir, { recursive: true, force: true });
   });
 
+  afterEach(async () => {
+    await rm(changesDir, { recursive: true, force: true });
```

Comment:
It would be cleaner to just delete the temporary branch that is used inside the test in the `afterEach()` instead of manual removal/addition of a folder.

Reply:
done

Reply:
Did you push all the changes? It still removes the directory instead of removing the branch. I added a snippet of what I mean below.

Reply:
Now i pushed, but i was not able to remove the line. added a todo